Dockerfile

```diff
@@ -1,39 +1,54 @@
 # Builder image that builds all the multy-back and all node services
 # multyio/multy-back-builder has all dependencies cached
+# Based on golang:1.9.4
```

Comment:
Link to Dockerfile for `multy-back-builder`

Reply:
It is now committed as separate Dockerfile, see `Dockerfile_multy-back-builder` and corresponding make targets:

```
docker-build-builder-image
docker-retag-builder-image
docker-push-builder-image
```